src/linear_algebra/finite_dimensional.lean

```diff
@@ -330,6 +330,22 @@ iff.trans (by { rw ← findim_eq_dim, norm_cast }) (@dim_pos_iff_nontrivial K V
 lemma findim_pos [finite_dimensional K V] [h : nontrivial V] : 0 < findim K V :=
 findim_pos_iff.mpr h
 
+/-- A finite dimensional space has zero `findim` iff it is a subsingleton.
+This is the `findim` version of `dim_zero_iff`. -/
+lemma findim_zero_iff [finite_dimensional K V] :
+  findim K V = 0 ↔ subsingleton V :=
+begin
+  rw [← not_iff_not, ← not_nontrivial_iff_subsingleton, not_not],
```

Comment:
Can this proof be replaced with something like
```
iff.trans (by { rw ← findim_eq_dim, norm_cast }) (@dim_zero_iff K V _ _ _)
```

Reply:
Done!